app-utils/screen/autobuild/defines

```diff
@@ -3,12 +3,15 @@ PKGSEC=utils
 PKGDEP="ncurses linux-pam libutempter"
 PKGDES="Full-screen window manager that multiplexes a physical terminal"
 
-AUTOTOOLS_AFTER="--enable-socket-dir \
-                 --enable-pam \
-                 --enable-use-locale \
+# Note: PTY group ID 5: tty.
+#
+# FIXME: --with-pty-rofs=no, no is upstream default, I have no idea what this
```

Comment:
> AH_TEMPLATE([PTY_ROFS], [pty devices are on read only filesystem])

<https://git.savannah.gnu.org/cgit/screen.git/tree/src/configure.ac?id=4c78830855281a82acc9dd6ab65c74c7320b7e68#n42>

Reply:
Thanks, I have revised the comment with more details.